src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { Form, FormGroup } from '@patternfly/react-core';
+
+import { useAppDispatch, useAppSelector } from '../../../../../store/hooks';
+import {
+  selectUserName,
+  selectUserPassword,
+  setUserNameByIndex,
+  setUserPasswordByIndex,
+} from '../../../../../store/wizardSlice';
+import { HookValidatedInput } from '../../../ValidatedTextInput';
+const UserInfo = () => {
+  const dispatch = useAppDispatch();
+  // eslint-disable-next-line react-redux/useSelector-prefer-selectors
+  const userNameSelector = selectUserName(0);
+  const userName = useAppSelector(userNameSelector);
+  const userPasswordSelector = selectUserPassword(0);
+  const userPassword = useAppSelector(userPasswordSelector);
+
+  const handleNameChange = (
+    _e: React.FormEvent<HTMLInputElement>,
+    value: string
+  ) => {
+    const index = 0;
```

Comment:
Even though this is temporary, we can DRY the code out by moving this outside of this handler and re-using it in handlePasswordChange.

Reply:
We can also re-use it in userNameSelector and userPasswordSelector. Right now the 0 we specify is sort of a "magic number", and it is a good practice to avoid those.